js/script.js

```diff
@@ -0,0 +1,98 @@
+const formInputs = document.querySelectorAll(".form__input-box");
+const formEmail = document.querySelector("#form__email");
+const formPassword = document.querySelector("#form__password");
+const formSubmit = document.querySelector(".form__submit");
+
+// error 종류에 따라 메시지 추가
+const errorMsgs = {
+  emptyInput: {
+    email: "이메일을 입력해주세요.",
+    password: "비밀번호를 입력해주세요.",
+    text: "비밀번호를 입력해주세요.",
+  },
+  invalidInput: {
+    email: "올바른 이메일 주소가 아닙니다.",
+  },
+  invalidLogin: {
+    email: "이메일을 확인해주세요.",
+    password: "비밀번호를 확인해주세요.",
+  },
+};
+
+function printErrorMsg(error, type, target) {
+  removeErrorMsg(target);
+  const errorMsg = document.createElement("p");
+  errorMsg.textContent = errorMsgs[error][type];
+  errorMsg.classList.add("error-msg");
+  target.after(errorMsg);
+}
+
+function removeErrorMsg(target) {
+  if (
+    target.nextElementSibling &&
+    target.nextElementSibling.classList.contains("error-msg")
+  ) {
+    target.nextElementSibling.remove();
+  }
+}
+
+// input에 값이 존재하는지 확인
+function checkEmptyInput(event) {
+  if (!event.target.value) {
+    event.target.classList.add("form__input-box__error");
+    printErrorMsg("emptyInput", event.target.type, event.target);
```

Comment:
printErrorMsg 함수의 parameter 가 많아지고 있는 것 같아요!
이럴 때는 named parameter 를 사용해주시는 것도 좋습니다.

예를 들어 아래와 같이 작성하는 것을 named parameter 라고 하는데요! 이렇게 하면 함수 호출 시, 매개변수의 이름을 알 수 있기 때문에 인수의 의미를 바로 이해할 수 있고, 함수를 직접 보지않아도 인터페이스를 호출부에서 바로 알 수 있어요.
```suggestion
    printErrorMsg({
      errorType: "emptyInput", 
      eventType: event.target.type, 
      target: event.target
    });
```